### PR TITLE
Test-177104 Selenium: update Parents/Children selector

### PR DESCRIPTION
**Pull request by IvLarionov**

### CODE REVIEW

File: src/main/java/com/onevizion/uitest/api/helper/RelationSelector.java [hunk 1]
```diff
@@ -138,7 +138,6 @@ public void checkCurrentValueInRelationSelector(Long gridIdx, String label) {
 
     private void waitRelationSelector(Long gridIdx) {
         relationSelectorWait.waitIsReadyRelationSelector(gridIdx);
-        relationSelectorWait.waitIsReadyMutationObserver(gridIdx);
```
Comment (kovalev007) [inline src/main/java/com/onevizion/uitest/api/helper/RelationSelector.java:141]: ?

File: src/main/java/com/onevizion/uitest/api/helper/RelationSelector.java [hunk 2]
```diff
@@ -128,7 +125,10 @@ public void chooseParentChildTrackorType(Long gridIdx, String trackorType) {
 
     public void checkCurrentValueInRelationSelector(Long gridIdx, String label, int count) {
         String actualLabel = seleniumSettings.getWebDriver().findElement(By.id(REL_SEL_MAIN_ELEMENT + gridIdx)).findElement(By.className(REL_SEL_MAIN_ELEMENT_CURRENT_TEXT)).getText();
-        Assert.assertEquals(actualLabel, label + count);
+        String actualCount = seleniumSettings.getWebDriver().findElement(By.id(REL_SEL_MAIN_ELEMENT + gridIdx)).findElement(By.className(REL_SEL_MAIN_ELEMENT_CURRENT_COUNT)).getText();
+        int intActualCount = Integer.parseInt(actualCount);
```
Comment (kovalev007) [inline src/main/java/com/onevizion/uitest/api/helper/RelationSelector.java:129]: String to Integer?
Assert.assertEquals(actualLabel + actualCount, label + count);

File: src/main/java/com/onevizion/uitest/api/helper/RelationSelector.java [hunk 3]
```diff
@@ -49,21 +50,17 @@
     private RelationSelectorJs relationSelectorJs;
 
     public void checkRelationSelectorValuesCount(Long gridIdx, int count) {
-        List<WebElement> rowNames = seleniumSettings.getWebDriver().findElement(By.id(REL_SEL_CONTAINER + gridIdx)).findElements(By.className(REL_SEL_CONTAINER_ITEM));
+        List<WebElement> rowNames = seleniumSettings.getWebDriver().findElement(By.className(REL_SEL_CONTAINER)).findElements(By.className(REL_SEL_CONTAINER_ITEM));
```
Comment (kovalev007) [inline src/main/java/com/onevizion/uitest/api/helper/RelationSelector.java:53]: gridIdx in all voids - tests will be broken without gridIdx. You can try open grid on 1, 2 ,3 ,4 tabs and execute code for any grid except 1 - tests will be failed.

Comment (IvLarionov) [reply src/main/java/com/onevizion/uitest/api/helper/RelationSelector.java]: Done

Comment (kovalev007) [reply src/main/java/com/onevizion/uitest/api/helper/RelationSelector.java]: We have two ways:
1) change names in tests
or
2) add/change IDS in system

**Pull request merged**